src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/domain/VedtaksperiodeTilsynBarnMapper.kt

```diff
@@ -36,11 +36,11 @@ object VedtaksperiodeTilsynBarnMapper {
             validatePeriode()
         }
 
-        constructor(stønadsperiode: StønadsperiodeBeregningsgrunnlag, barn: List<BarnId>) : this(
-            fom = stønadsperiode.fom,
-            tom = stønadsperiode.tom,
-            målgruppe = stønadsperiode.målgruppe,
-            aktivitet = stønadsperiode.aktivitet,
+        constructor(vedtakpseriodeBeregningsgrunnlag: VedtaksperiodeBeregningsgrunnlag, barn: List<BarnId>) : this(
```

Comment:
Blir litt kortere hvis man kun kaller den for vedtaksperiode
```suggestion
        constructor(vedtaksperiode: VedtaksperiodeBeregningsgrunnlag, barn: List<BarnId>) : this(
```